submissions/AriadnaKravchuk/memory-game/script.js

```diff
@@ -0,0 +1,179 @@
+const body = document.querySelector("body");
+const startPage = document.querySelector(".start-page");
+const gamePage = document.querySelector(".game-page");
+const popup = document.querySelector(".popup");
+const allDetails = document.querySelectorAll(".details");
+const detailsButtons = document.querySelectorAll(".details__button");
+const popupButtons = document.querySelectorAll(".popup__button");
+const cards = document.querySelectorAll(".card");
+const cardsInner = document.querySelectorAll(".card__inner");
+const cardsBacks = document.querySelectorAll(".card__back");
+const cardsFronts = document.querySelectorAll(".card__front");
+const cardsIcons = document.querySelectorAll(".card__img");
+const gameTime = document.querySelector(".game__time");
+const gameMoves = document.querySelector(".game__moves");
+const scorePopup = document.querySelector(".popup__score");
+
+let theme = "",
+    matches = 0,
+    moves = 0;
+let isEndGame = false,
+    secondsCounter = 0,
+    minutesCounter = 0,
+    minutes = "00",
+    seconds = "00";
+let firstCard = [],
+    secondCard = [],
```

Comment:
Why do you use `array` as default or empty value for this variables? You don't use array in dependence of variables logic.